### ISSUE TITLE
Update Vue to enable multi-select filters

### ISSUE THREAD

**Issue by scottveirs** [salish-sea/acartia#63]
@robert-ohurley and team flagged that current version of Vue is limiting ability to filter data (e.g. select both orcas and humpbacks from a species filter). From [September progress call notes](https://docs.google.com/document/d/1bfjEtBxcLPu6U2LCBdK6WY_-gz3Q3j4qcKEeUulWkTI/edit?tab=t.0):

```Multiselect is not particularly viable with such an old version of Vue. The few available options are either not compatible or won’t be supported for updates. Swinburne team is still investigating multi select filters however due to component library incompatibilities with vue 2, this is difficult.```

**Comment by scottveirs** [comment]
@robert-ohurley I gather from our subsequent calls and the current beta site that you all surmounted this issue. 

Do you and @aalaydrus  agree that it can now be closed?

**Comment by robert-ohurley** [comment]
Yup, this issue can be closed. We upgraded the application from Vue 2 to Vue 3. Most libraries will be compatible with this moving forward, which will make it easier to extend the application in the future. The only downside being that the MDBVue library that had been used to initially develop the app is incompatible. Remaining components in use using this (if any) will need to be updated.
